ui/lib/apps/TopSQL/utils/othersRecord.ts

```diff
@@ -0,0 +1,29 @@
+import { TopsqlCPUTimeItem, TopsqlPlanItem } from '@lib/client'
+import { PlanRecord } from '../pages/List/ListDetail/ListDetailTable'
+import { SQLRecord } from '../pages/List/ListTable'
+
+const OTHERS_LABEL = '(Others)'
+
+export function convertOthersRecord(r: TopsqlCPUTimeItem) {
+  if (!!r.sql_digest) {
+    return
+  }
+  r.sql_digest = OTHERS_LABEL
+  r.sql_text = OTHERS_LABEL
+}
```

Comment:
How about adding a field called `isOthers`? Then you don't need utility functions in this file at all.

Reply:
Great!